opennebula/resource_opennebula_template_test.go

```diff
@@ -216,6 +216,10 @@ resource "opennebula_template" "template" {
   memory = "512"
   description = "Template created for provider acceptance tests"
 
+  features = {
```

Comment:
This only ensure that the provider know the `features` via it's schema.
If the `features` section was read by the provider, we could also test it's value to ensure that the apply is accurate